packages/block-editor/src/components/iframe/index.js

```diff
@@ -1,164 +1,39 @@
-/**
```

Comment:
For debugging purposes I replaced this entire file with the implementation that I managed to get working here https://codesandbox.io/s/rendering-emotion-iframe-into-body-forked-d9qyx?file=/src/index.js